docs/data/data-grid/localization/localization.md

```diff
@@ -17,6 +17,22 @@ In the following example, the labels of the density selector are customized.
 
 {{"demo": "CustomLocaleTextGrid.js", "bg": "inline"}}
 
+> ⚠️ It's important to note that because the data grid uses components from the Material UI Core library some translation keys need to be accessed using that component key.
```

Comment:
I'm not aware we ever reference it this way

```suggestion
> ⚠️ It's important to note that because the data grid uses components from the Material UI library some translation keys need to be accessed using that component key.
```

Reply:
Also, I think that we should use specialized callouts https://github.com/mui/material-ui/pull/32402. 

@siriwatknp did we ever announce the change to the other teams? I believe we didn't, maybe we should?

Reply:
https://github.com/mui/mui-x/pull/5008

Reply:
So the correct one we should use it `:::info`?

Reply:
> I'm not aware we ever reference it this way

We never explicitly said it. I think the initial assumption was that people will use the theme to overwrite those texts but if you want to use the `localeText` only then this is the way.